inference-engine/tests/functional/shared_test_classes/src/single_layer/softmax.cpp

```diff
@@ -23,31 +25,38 @@ std::string SoftMaxLayerTest::getTestCaseName(testing::TestParamInfo<softMaxLaye
     result << "inL=" << inLayout << "_";
     result << "outL=" << outLayout << "_";
     result << "IS=" << CommonTestUtils::vec2str(inputShape) << "_";
+    result << "TS=" << CommonTestUtils::vec2str(targetShapes) << "_";
     result << "axis=" << axis << "_";
     result << "trgDev=" << targetDevice;
 
     return result.str();
 }
 
 void SoftMaxLayerTest::SetUp() {
-    InferenceEngine::SizeVector inputShape;
-    InferenceEngine::Precision netPrecision;
-    size_t axis;
+    std::vector<std::pair<size_t, size_t>> inputShape;
+    std::vector<InferenceEngine::SizeVector> targetShapes;
 
-    std::tie(netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, axis, targetDevice, configuration) = GetParam();
+    std::tie(netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetShapes, axis, targetDevice, configuration) = GetParam();
     outLayout = inLayout;
 
-    const auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    for (auto&& targetShape : targetShapes) {
+        targetStaticShapes.emplace_back(targetShape);
+    }
 
-    const auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+    inputDynamicShape = FuncTestUtils::PartialShapeUtils::vec2partialshape(inputShape, targetStaticShapes[0]);
+
+    makeSoftMaxFunction();
+}
 
+void SoftMaxLayerTest::makeSoftMaxFunction() {
```

Comment:
please rename the function according the code style and move it to the [builders](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/builders.hpp) and [here](https://github.com/openvinotoolkit/openvino/tree/master/inference-engine/tests/ngraph_helpers/ngraph_functions/src)

Reply:
It's changed to makeSoftMax(), and could you let me know if you have any other suggestion or comment?